drop-in/src/main/res/values/styles.xml

```diff
@@ -51,8 +51,9 @@
         <item name="android:paddingRight">@dimen/standard_margin</item>
         <item name="android:textAllCaps">true</item>
         <item name="android:textIsSelectable">false</item>
-        <item name="android:textSize">14sp</item>
+        <item name="android:textSize">12sp</item>
```

Comment:
WDYT about deprecating this style and creating a new one? If anyone is currently overriding this style they will not get a nice result since these labels are now subtitles instead of titles